Twisted-Tutorial.ipynb

```diff
@@ -262,6 +403,15 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We wrap the service in a function that looks local -- almost. It does return a deferred, since it can block for a long time (for example, if the addition service is overloaded). This is a typical pattern: we can often find high level wrappers to isolate the details of the remote protocol, but the essential fact that it is remote, and can be slow (and potentially fail) remains.\n",
```

Comment:
This is a really important point that I think deserves to be expanded on.  What you mean by "local" should be made clear to the point of redundancy, and you should say "service" instead of "protocol" to remain consistent.  Something maybe like:

We'll wrap calls to our new addition service in a function so that invoking the service appears to be a local, in process action -- almost. The wrapper will return a Deferred, since it can block for a long time (for example, if the addition service is overloaded). This is a typical pattern: Twisted code often includes high-level wrappers that encapsulate the details of talking to a remote service, but the fact remains that the service is remote, and calls to it can be slow and even fail.